### ISSUE TITLE
[FEAT] Add option to use crowdsourced street views API (Mapillary / KartaView)

### ISSUE THREAD

**Issue by SmileyAG** [GeoGuess/Geoguess#197]
**Is your feature request related to a problem? Please describe.**
We could avoid stuff like in #158 issue. I don't want to specify billing address and etc for usage Google Street View API.

**Comment by BurAndBY** [comment]
There is a project KartaView. 
https://wiki.openstreetmap.org/wiki/KartaView

**Comment by BurAndBY** [comment]
All of the services dont have even photosphere

**Comment by SmileyAG** [comment]
> 
> 
> All of the services dont have even photosphere
 
Wrong. You could specify in Mapillary to show 360° panoramas only. 

Examples of some Mapillary based implementations:

https://github.com/brownr46/geoguesser (https://geoguesser-ca064.web.app)
https://github.com/hugisveitti/hugiholmcom/tree/master/map-game (https://hugiholm.herokuapp.com/jeojuessr)

**Comment by BurAndBY** [comment]
I see. I'll try that with Mapillary and OpenStreetViee.

**Comment by BilelJegham** [comment]
Yes, I will be great to have a complete demo version unlimited without Google quota

Thx @SmileyAG for Github repos
